DevelopedCalculator.js

```diff
@@ -30,6 +30,12 @@ class DevelopedCalculator extends Calculator {
         //onClearButtonClick() {
         //    super.onClearButtonClick()
         // }
+    onEqualButtonClick() {
```

Comment:
is this working now?

Reply:
No it’s not 
I have commented that there is a bug